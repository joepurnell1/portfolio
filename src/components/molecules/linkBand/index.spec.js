import React from 'react';
import renderer from 'react-test-renderer';
import LinkBand from './index';

describe('link band component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <LinkBand />,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
