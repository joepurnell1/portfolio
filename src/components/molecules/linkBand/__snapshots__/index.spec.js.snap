// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link band component should render default as expected 1`] = `
<div
  className="sc-ifAKCX dKGVbF"
>
  <a
    className="sc-bdVaJa iXjBRK"
    href="https://github.com/joepurnell1"
    size="45px"
  >
    <img
      alt="github"
      className="sc-bwzfXH bLQhUk"
      height="40px"
      src="test-file-stub"
      width="40px"
    />
  </a>
  <a
    className="sc-bdVaJa iXjBRK"
    href="https://twitter.com/joepurnell8?lang=en"
    size="45px"
  >
    <img
      alt="twitter"
      className="sc-bwzfXH bLQhUk"
      height="40px"
      src="test-file-stub"
      width="40px"
    />
  </a>
  <a
    className="sc-bdVaJa iXjBRK"
    href="https://www.linkedin.com/in/joe-purnell-7294295b/"
    size="45px"
  >
    <img
      alt="linkedin"
      className="sc-bwzfXH bLQhUk"
      height="40px"
      src="test-file-stub"
      width="40px"
    />
  </a>
</div>
`;
